java/runtime/src/main/java/org/ray/runtime/serializer/FSTSerializer.java

```diff
@@ -7,19 +7,17 @@
 /**
  * Java object serialization TODO: use others (e.g. Arrow) for higher performance
  */
-public class Serializer {
-
+// FST is much better than Fst, e.g. FSTConfiguration
```

Comment:
FST library use FST prefix. I think Fst is worse choice.

Reply:
FST library doesn't follow google style, so it's okay for them to use `FST`. But since we're following google style, it's important to be consistent across our code base. By they way, only upper-casing the first letter is a very common naming convention in Java, e.g., there are dozens of `HttpXxx` classes in JDK.

Reply:
OK. I will rename the `FST` to `Fst`.